### ISSUE TITLE
Pedestrian routing does not use highway=bridleway

### ISSUE THREAD

**Issue by JesseCrocker** [valhalla/valhalla#1076]
TL;DR: default access on highway=bridleway does not allow pedestrians, but it should, and did untill recently.

For information on server setup and software version see https://github.com/valhalla/valhalla/issues/1070

This request fails on build from master last week
```
curl 'https://routing.gaiagps.com/route?json=\{%22locations%22:\[\{%22lat%22:36.68429040751335,%22lon%22:-118.76705646514893,%22type%22:%22break%22\},\{%22lat%22:36.68282773449172,%22lon%22:-118.76582264900206,%22type%22:%22break%22\}\],%22costing%22:%22pedestrian%22\}' | jq .
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  1206  100  1206    0     0   1531      0 --:--:-- --:--:-- --:--:--  1530
{
  "trip": {
    "language": "en-US",
    "summary": {
      "max_lon": -118.765694,
      "max_lat": 36.682701,
      "time": 1,
      "length": 0.002,
      "min_lat": 36.682693,
      "min_lon": -118.765709
    },
    "locations": [
      {
        "side_of_street": "right",
        "lon": -118.767059,
        "lat": 36.684292,
        "type": "break"
      },
      {
        "side_of_street": "right",
        "lon": -118.765823,
        "lat": 36.682827,
        "type": "break"
      }
    ],
    "units": "kilometers",
    "legs": [
      {
        "shape": "y{|}dAzf{oaFN\\",
        "summary": {
          "max_lon": -118.765694,
          "max_lat": 36.682701,
          "time": 1,
          "length": 0.002,
          "min_lat": 36.682693,
          "min_lon": -118.765709
        },
        "maneuvers": [
          {
            "travel_type": "foot",
            "time": 1,
            "type": 2,
            "verbal_multi_cue": true,
            "verbal_pre_transition_instruction": "Walk southwest for less than 10 meters. Then Your destination will be on the right.",
            "instruction": "Walk southwest.",
            "end_shape_index": 1,
            "length": 0.002,
            "begin_shape_index": 0,
            "rough": true,
            "travel_mode": "pedestrian"
          },
          {
            "travel_type": "foot",
            "travel_mode": "pedestrian",
            "begin_shape_index": 1,
            "time": 0,
            "type": 5,
            "end_shape_index": 1,
            "instruction": "Your destination is on the right.",
            "length": 0,
            "verbal_transition_alert_instruction": "Your destination will be on the right.",
            "verbal_pre_transition_instruction": "Your destination is on the right."
          }
        ]
      }
    ],
    "status_message": "Found route between points",
    "status": 0
  }
}
```

But works on valhalla.mapzen.com
```
curl 'https://valhalla.mapzen.com/route?json=\{%22locations%22:\[\{%22lat%22:36.68429040751335,%22lon%22:-118.76705646514893,%22type%22:%22break%22\},\{%22lat%22:36.68282773449172,%22lon%22:-118.76582264900206,%22type%22:%22break%22\}\],%22costing%22:%22pedestrian%22\}&api_key=search-DAC_OdA'  | jq .
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  1141  100  1141    0     0   1552      0 --:--:-- --:--:-- --:--:--  1554
{
  "trip": {
    "language": "en-US",
    "summary": {
      "max_lon": -118.765831,
      "max_lat": 36.684273,
      "time": 165,
      "length": 0.235,
      "min_lat": 36.682819,
      "min_lon": -118.767067
    },
    "locations": [
      {
        "lon": -118.767059,
        "lat": 36.684292,
        "type": "break"
      },
      {
        "lon": -118.765823,
        "lat": 36.682827,
        "type": "break"
      }
    ],
    "units": "kilometers",
    "legs": [
      {
        "shape": "_~_~dAt|}oaFlDuYfDkAnI}IvDmKnD|@xLePfC{A`DjBbF\\xBeEl@jAnDl@~CxBdJoId@}@",
        "summary": {
          "max_lon": -118.765831,
          "max_lat": 36.684273,
          "time": 165,
          "length": 0.235,
          "min_lat": 36.682819,
          "min_lon": -118.767067
        },
        "maneuvers": [
          {
            "travel_type": "foot",
            "travel_mode": "pedestrian",
            "rough": true,
            "begin_shape_index": 0,
            "length": 0.235,
            "end_shape_index": 15,
            "instruction": "Walk east.",
            "verbal_pre_transition_instruction": "Walk east for 200 meters.",
            "type": 1,
            "time": 165
          },
          {
            "travel_type": "foot",
            "travel_mode": "pedestrian",
            "begin_shape_index": 15,
            "time": 0,
            "type": 4,
            "end_shape_index": 15,
            "instruction": "You have arrived at your destination.",
            "length": 0,
            "verbal_transition_alert_instruction": "You will arrive at your destination.",
            "verbal_pre_transition_instruction": "You have arrived at your destination."
          }
        ]
      }
    ],
    "status_message": "Found route between points",
    "status": 0
  }
}
```

Way that it wont route on https://www.openstreetmap.org/way/35867367
This way has 1 tag, highway=bridleway

According to https://wiki.openstreetmap.org/wiki/OSM_tags_for_routing/Access-Restrictions The default access for highway=bridleway in the USA allows for pedestrians.

Info for this way from locate
Queried from https://trailbehind.github.io/valhalla-demos/locate/#17/36.68227/-118.76491
```
[
  {
    "edge_id": {
      "id": 1278,
      "value": 42888395170,
      "tile_id": 728884,
      "level": 2
    },
    "edge_info": {
      "shape": "_}|}dAnd{oaFdPm_@tTs[dJuN",
      "way_id": 358673674,
      "names": []
    },
    "edge": {
      "classification": {
        "link": false,
        "internal": false,
        "surface": "compacted",
        "classification": "service_other"
      },
      "end_node": {
        "id": 529,
        "value": 17756125602,
        "tile_id": 728884,
        "level": 2
      },
      "speed": 10,
      "traffic_signal": false,
      "start_restriction": {
        "moped": false,
        "wheelchair": false,
        "taxi": false,
        "HOV": false,
        "truck": false,
        "emergency": false,
        "pedestrian": false,
        "car": false,
        "bus": false,
        "bicycle": false
      },
      "speed_limit": 0,
      "geo_attributes": {
        "weighted_grade": 0,
        "length": 142
      },
      "cycle_lane": "none",
      "access_restriction": false,
      "part_of_complex_restriction": false,
      "country_crossing": false,
      "has_exit_sign": false,
      "lane_count": 1,
      "speed_type": "classified",
      "drive_on_right": false,
      "destination_only": false,
      "seasonal": false,
      "tunnel": false,
      "bridge": false,
      "access": {
        "moped": false,
        "wheelchair": false,
        "taxi": false,
        "HOV": false,
        "truck": false,
        "emergency": false,
        "pedestrian": false,
        "car": false,
        "bus": false,
        "bicycle": false
      },
      "toll": false,
      "round_about": false,
      "bike_network": {
        "mountain": false,
        "local": false,
        "regional": false,
        "national": false
      },
      "end_restriction": {
        "moped": false,
        "wheelchair": false,
        "taxi": false,
        "HOV": false,
        "truck": false,
        "emergency": false,
        "pedestrian": false,
        "car": false,
        "bus": false,
        "bicycle": false
      },
      "unreachable": false,
      "forward": true,
      "not_thru": false,
      "truck_route": false,
      "use": "bridleway"
    },
    "minimum_reachability": -1,
    "score": 34.3,
    "traffic_segments": [],
    "percent_along": 0.5899,
    "correlated_lon": -118.764908,
    "side_of_street": "neither",
    "correlated_lat": 36.68227
  },
  {
    "edge_id": {
      "id": 1282,
      "value": 43022612898,
      "tile_id": 728884,
      "level": 2
    },
    "edge_info": {
      "shape": "_}|}dAnd{oaFdPm_@tTs[dJuN",
      "way_id": 358673674,
      "names": []
    },
    "edge": {
      "classification": {
        "link": false,
        "internal": false,
        "surface": "compacted",
        "classification": "service_other"
      },
      "end_node": {
        "id": 527,
        "value": 17689016738,
        "tile_id": 728884,
        "level": 2
      },
      "speed": 10,
      "traffic_signal": false,
      "start_restriction": {
        "moped": false,
        "wheelchair": false,
        "taxi": false,
        "HOV": false,
        "truck": false,
        "emergency": false,
        "pedestrian": false,
        "car": false,
        "bus": false,
        "bicycle": false
      },
      "speed_limit": 0,
      "geo_attributes": {
        "weighted_grade": 0,
        "length": 142
      },
      "cycle_lane": "none",
      "access_restriction": false,
      "part_of_complex_restriction": false,
      "country_crossing": false,
      "has_exit_sign": false,
      "lane_count": 1,
      "speed_type": "classified",
      "drive_on_right": false,
      "destination_only": false,
      "seasonal": false,
      "tunnel": false,
      "bridge": false,
      "access": {
        "moped": false,
        "wheelchair": false,
        "taxi": false,
        "HOV": false,
        "truck": false,
        "emergency": false,
        "pedestrian": false,
        "car": false,
        "bus": false,
        "bicycle": false
      },
      "toll": false,
      "round_about": false,
      "bike_network": {
        "mountain": false,
        "local": false,
        "regional": false,
        "national": false
      },
      "end_restriction": {
        "moped": false,
        "wheelchair": false,
        "taxi": false,
        "HOV": false,
        "truck": false,
        "emergency": false,
        "pedestrian": false,
        "car": false,
        "bus": false,
        "bicycle": false
      },
      "unreachable": false,
      "forward": false,
      "not_thru": false,
      "truck_route": false,
      "use": "bridleway"
    },
    "minimum_reachability": -1,
    "score": 34.3,
    "traffic_segments": [],
    "percent_along": 0.4101,
    "correlated_lon": -118.764908,
    "side_of_street": "neither",
    "correlated_lat": 36.68227
  }
]
```

**Comment by gknisely** [comment]
Hi @JesseCrocker 
Thank you for reporting this issue.  I am not sure what is going on here as we have not changed anything with regards to permissions for awhile.  I will dig into this as soon as I can do so.  
Thanks.  --Greg.

**Comment by dnesbitt61** [comment]
@JesseCrocker - this works for me locally. Bridleways do trigger the "country specific" access logic to determine if pedestrians have access. Are you setting up the admin database and making sure it is used?

**Comment by dnesbitt61** [comment]
@JesseCrocker - more info here: https://github.com/valhalla/valhalla/blob/master/docs/mjolnir/getting_started_guide.md

**Comment by JesseCrocker** [comment]
Thats the problem, I didnt build the admin areas.
